src/test/steps/happy-path.ts

```diff
@@ -177,9 +177,6 @@ Given("I've completed all happy path questions correctly", async () => {
   iClick('Continue');
 
   I.waitInUrl('/dividing-money-property');
-  iClick('Continue');
```

Comment:
happy path still needs the pages tho right? and a no

Reply:
We've moved the form from `/do-you-want-to-apply-financial-order` to `/dividing-money-property` and dropped the `/do-you-want-to-apply-financial-order` page from the user journey. As that financial order page doesn't exist anymore, the happy path now is to click `No` on `/dividing-money-property` and then continue straight on to `/upload-your-documents`